parts_list/extra_parts.md

```diff
@@ -16,14 +18,23 @@ You'll want these to mount the PCBs to the rover body. Many options exist here,
 
 We recommend getting spools of 18AWG (red and black), 20AWG (white, red, black) and 22AWG wire (4 colors). The wire has to be stranded and not solid core as it will bend during operation. For each wire type, if possible, get ribbon cable where each wire is attached to another one which will help with keeping wiring clean and manageable.
 
+Links for the wire can be found below on Amazon:
+
+- [18AWG](https://www.amazon.com/Silicone-Electrical-Conductor-Parallel-Flexible/dp/B07FMRDP87/)
+
+- [20AWG](https://www.amazon.com/Gauge-Electrical-Wire-Conductor-Cable/dp/B0B3MGD254)
```

Comment:
I wouldn't pick an option that is shielded and instead use ribbon cable. While this works, I would be worried that the shielding wouldn't fit through the body axis along with the other cables.